CMakeLists.txt

```diff
@@ -155,11 +130,59 @@ function(define_module_target MOD_NAME MOD_SRC MOD_NVCC_FLAGS)
   target_compile_options(${MOD_NAME} PRIVATE
     $<$<COMPILE_LANGUAGE:${CUDA_LANG}>:${MOD_NVCC_FLAGS}>)
   target_compile_definitions(${MOD_NAME} PRIVATE "-DTORCH_EXTENSION_NAME=${MOD_NAME}")
-  target_include_directories(${MOD_NAME} PRIVATE csrc PRIVATE ${TORCH_INCLUDE_DIRS} ${MPI_CXX_INCLUDE_DIRS})
-  target_link_libraries(${MOD_NAME} PRIVATE ${TORCH_LIBRARIES})
+  target_include_directories(${MOD_NAME} PRIVATE csrc PRIVATE ${MPI_CXX_INCLUDE_DIRS})
+  target_link_libraries(${MOD_NAME} PRIVATE ${TORCH_LIBRARIES}) # this also sets include dirs
```

Comment:
If you look at `lib/python3.10/site-packages/torch/share/cmake/Torch/TorchConfig.cmake`, the `torch` target (which is in `TORCH_LIBRARIES`) had `include_dirs` set on it.